### PR TITLE
Add Ubuntu Focal.

### LINKED ISSUE #8195: Add Focal to PPA scripts
Add Ubuntu Focal (2020-04) to our ppa scripts and test that it is working.

### PR DESCRIPTION
**Pull request by chriseth**
Closes #8195

### CODE REVIEW

**Comment by chriseth** [comment]
Just ran the focal z3 script: https://launchpad.net/~ethereum/+archive/ubuntu/cpp-build-deps/+build/18698203

File: scripts/install_deps.sh [hunk 1]
```diff
@@ -300,7 +300,8 @@ case $(uname -s) in
                         install_z3="libz3-dev"
                         ;;
                     bionic)
-                        echo "Installing solidity dependencies on Ubuntu Bionic (18.04)."
+                    focal)
+                        echo "Installing solidity dependencies."
                         install_z3="libz3-dev"
```
Comment (ekpyron) [inline scripts/install_deps.sh]: This one won't work, that'd be the stock ubuntu libz3 which is too old... Actually I'm generally not sure this entire script works all that well these days...

Comment (ekpyron) [reply scripts/install_deps.sh]: Actually wrong - focal actually has the right one I just noticed! It won't work for bionic above, though :-D.

**Comment by ekpyron** [comment]
Since ubuntu focal actually has libz3-dev 4.8.7, I think we actually don't need the static Z3 PPA anymore for it and we should rather change the release PPA script from installing ``libz3-static-dev`` to installing ``libz3-dev``, **if** on focal...

**Review by ekpyron — CHANGES_REQUESTED** [review]
We should try building against stock ``libz3-dev`` instead of our ``libz3-static-dev`` for focal.

**Comment by chriseth** [comment]
Saw the comment too late. Ran the release_ppa.sh script to see if it works.

**Comment by ekpyron** [comment]
New build against stock Z3 triggered here: https://launchpad.net/~ethereum/+archive/ubuntu/ethereum-dev/+build/18704560

**Review by ekpyron — APPROVED** [review]

**Pull request merged**